sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2593,19 +2597,27 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
         restrict: 'A',
         scope: {},
         template: [
-            '<div class="row text-center">',
-              '<p>Enter your email address and we\'ll send an authorization link to your inbox.</p>',
+            '<div data-ng-show="isJupyterhub" class="alert alert-info col-sm-offset-2 col-sm-10" role="alert">',
+            'We’re improving your Jupyter experience by making both Jupyter and Sirepo accessible via a single email login. Simply follow the directions below to complete this process.',
             '</div>',
             '<form class="form-horizontal" autocomplete="off" novalidate>',
-              '<div class="row text-center">',
-                '<label class="col-sm-3 control-label">Your Email</label>',
-                '<div class="col-sm-9">',
+              '<div class="form-group">',
+                '<div class="col-sm-offset-2 col-sm-10">',
+                  '<p>Enter your email address and we\'ll send an authorization link to your inbox.</p>',
+                '</div>',
+              '</div>',
+              '<div class="form-group">',
+                '<label class="col-sm-2 control-label">Your Email</label>',
+                '<div class="col-sm-10">',
                   '<input type="text" class="form-control" data-ng-model="data.email" required/>',
                   '<div class="sr-input-warning" data-ng-show="showWarning">{{ warningText }}</div>',
                 '</div>',
               '</div>',
-              '<div class="row text-center" style="margin-top: 10px">',
-                 '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+              '<div class="form-group">',
+                '<div class="col-sm-offset-2 col-sm-10">',
+                  '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+                  '<p class="help-block">By signing up for Sirepo you agree to Sirepo’s <a href="https://www.sirepo.com/en/privacy.html">privacy policy</a> and <a href="https://www.sirepo.com/en/terms.html">terms and conditions</a>, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.</p>',
```

Comment:
apostrophe vs single quote and no absolute uris.

Reply:
https://github.com/radiasoft/sirepo/pull/3232/commits/b152a032d3e8d5bea02db98e32c6c8f53223ad8c